package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "factorial-i18n",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
This is definitely a mayor version change :trollface:

Reply:
Also are we using now `numeral` for something? if not remove from dependencies

Reply:
It's still being used in `setLocale`, but I haven't investigated properly whether it still needs to be used. I can take a look if you think there's a big advantage in removing it from dependencies.

Reply:
Removed numeral 👍